ooniprobe/View/Informed consent/ICSwipeViewController.h

```diff
@@ -0,0 +1,22 @@
+#import <UIKit/UIKit.h>
+#import "Onboarding1ViewController.h"
+#import "Onboarding2ViewController.h"
+#import "Onboarding3ViewController.h"
+#import "Onboarding4ViewController.h"
```

Comment:
So, this is the view controller that you can swipe left or right to move around in onboarding?

Reply:
These are the single view controllers